### DISCUSSION TITLE
essay/dev/vitepress-plugins

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#309] [Announcements]
# essay/dev/vitepress-plugins

主题在近1年多的时间里集成了非常的多功能了，接下来计划将这些能力拆分成组件，以便于更方便的与其它主题集成。

https://sugarat.top/essay/dev/vitepress-plugins.html

**Comment by hacxy** [comment]
分页插件有计划了吗? 😎

**Reply by ATQQ** [reply]
文章列表嘛？

**Reply by hacxy** [reply]
> 文章列表嘛？

对的
